### PR TITLE
Feature/Partial CRUD implementation

### PR DESCRIPTION
**Pull request by Alejuda**
In this branch, we worked pair-to-pair we my coding partner @redmarverick and created the following:

- Add HTML, js, and CSS linters.
- Add index.html and index.js files.
- Add HTML structure to manipulate it from the JavaScript file.
- Add JavaScript logic to create and remove books.
- Fix linter errors.

### CODE REVIEW

**Review by joel-kalema — COMMENTED** [review]
Hi @Alejuda 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that has linters that does not have a descriptive README


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    description: desc,
+  };
+  books.push(newBook);
+};
+
+const removeBook = (id) => {
+  const remove = books.findIndex((book) => book.id === id);
+  books.splice(remove, 1);
+  localStorage.setItem('books-colection', JSON.stringify(books));
+};
```
Comment (AbelG101) [inline index.js:26]: - [ ] It seems like the `removeBook()` function is not working properly as the proper book is not being removed please check the below video: 👇
  firefox_lqMEtg1ed6
- As you can see from the above video "book 2" should've been removed but it's "book 1" that gets removed. 
- I've tried to track down the problem and it seems part of the problem is from the way you assign id's to each book from the `addBook()` function check the screenshot below:
  Inkedfirefox_MEL3jAEuqE
- As you can see from the above the `id` getting passed to the `removeBook()` is null and this is causing major problems so please make sure you fix it.✔

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Alejuda & @redmarverick  👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Alejuda) [reply index.js:35]: Thanks for your time! Your review was excellent.

**Review by Strangeal — APPROVED** [review]
Hi @Alejuda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**